src/app/api/gql/account/fragments/fragment-account-basics.graphql

```diff
@@ -2,4 +2,5 @@ fragment AccountBasics on Account {
   id
   accountName
   accountProvider
+  avatarUrl
```

Comment:
Should we get rid of AccountExtended? I only see 1 use left in the metadata block fragment.